test/win32.js

```diff
@@ -4,9 +4,11 @@ const config = require('./config.json')
 const { packager } = require('../dist')
 const path = require('path')
 const test = require('ava')
+const _ = require('lodash')
```

Comment:
We don't have `lodash` explicitly listed as a dev dep right now, should we add it?